action.yml

```
@@ -19,6 +19,9 @@ inputs:
   token:
     description: "Attic authorization token"
     required: false
+  skipAddingSubstituter:
```

I usually prefer kebab case for action inputs

```suggestion
  skip-adding-substituter:
```